src/Workspaces/Core/Portable/FindSymbols/FindReferences/Finders/AbstractReferenceFinder.cs

```diff
@@ -60,7 +60,7 @@ protected static async Task<ImmutableArray<Document>> FindDocumentsAsync(Project
             }
 
             var documents = ArrayBuilder<Document>.GetInstance();
-            foreach (var document in project.Documents)
+            foreach (var document in await project.GetAllRegularAndSourceGeneratedDocumentsAsync(cancellationToken).ConfigureAwait(false))
```

Comment:
so this is the true meat of the change.  i'm ok with this change here.  However, i think i'm going to go in after this to changup how this works.  Specifically, i think this shoul be an IAsyncEnumerable that prioritizes nont-generated docs, then retursn the generated docs.  That way we can get results in normal docs as soon as possible, and not block on all codegen.  then, the generated files can come later.

Reply:
Even though we could put the non-generated documents first, the moment you ask for semantics you'll still have to run generators to ensure you can even get accurate semantics in the non-generated docs. That said, I can imagine this is still a good win since you'll be able to get through syntax processing in parallel with other work.

Reply:
Yeah... that's a good point.  The value would be minimized there becaues of hte high semantic cost.  However, we do do syntactic/index searching in parallel.  So it would be good to get the set of documents to work on asap.